drf_writable_nested/mixins.py

```diff
@@ -115,6 +115,9 @@ def update_or_create_reverse_relations(self, instance, reverse_relations):
                 if related_data is None:
                     # Skip processing for empty data
                     continue
+                pk_name = field.Meta.model._meta.pk.attname
+                if pk_name not in related_data:
+                    related_data[pk_name] = instance.pk
```

Comment:
It seems the line ` related_data[pk_name] = instance.pk` should be replaced with something like that:
```
related_instance = getattr(instance, field_source, None)
if related_instance:
    related_data[pk_name] = related_instance.pk
```